Tests fail on new camb version

Two tests against precomputed likelihood values now fail see [here,](https://github.com/simonsobs/SOLikeT/actions/runs/4067655125/jobs/7422353830#step:5:395) I'm pretty sure due to the bump in camb version to v1.4.

e.g.
```
FAILED test_clusters.py::test_clusters_loglike - assert array([False])
 +  where array([False]) = (array([-854.8981993]), -855.0)
```

I can just update the test against the new precomputed likelihood values, or change the tolerance on the `isclose`.

--- a/soliket/tests/test_clusters.py
+++ b/soliket/tests/test_clusters.py
@@ -26,6 +26,7 @@ info_fiducial = {
                 "nonlinear": False,
                 "kmax": 10.0,
                 "dark_energy_model": "ppf",
+                "bbn_predictor": "PArthENoPE_880.2_standard.dat"
             }
         },
     },
@@ -43,7 +44,7 @@ def test_clusters_loglike():
 
     lnl = model_fiducial.loglikes({})[0]
 
-    assert np.isclose(lnl, -855.0)
+    assert np.isclose(lnl, -854.89406321)
 
 
 def test_clusters_n_expected():

--- a/soliket/tests/test_cross_correlation.py
+++ b/soliket/tests/test_cross_correlation.py
@@ -160,7 +160,7 @@ def test_shearkappa_hartlap(request):
     model = get_model(info)
     loglikes_hartlap, derived = model.loglikes()
 
-    assert np.isclose(np.abs(loglikes - loglikes_hartlap), 0.00104092)
+    assert np.isclose(np.abs(loglikes - loglikes_hartlap), 0.0010403)
 
 
 def test_shearkappa_deltaz(request):

